Shared/Samples/Download vector tiles to local cache/DownloadVectorTilesToLocalCacheView.swift

```diff
@@ -103,19 +103,19 @@ struct DownloadVectorTilesToLocalCacheView: View {
                                 )
                                 
                                 // Creates an envelope from the rectangle.
-                                guard let extent = mapView.envelope(fromViewRect: viewRect) else { return }
+                                guard let extent = mapViewProxy.envelope(fromViewRect: viewRect) else { return }
                                 
                                 // Downloads the vector tiles.
                                 do {
                                     try await model.downloadVectorTiles(extent: extent)
-                                    // Sets show results to true.
+                                    // Shows results when the download finishes.
                                     isShowingResults = true
                                     // Sets downloading to false when the download finishes.
                                     isDownloading = false
                                 } catch {
                                     // Shows an alert if any errors occur.
                                     self.error = error
-                                    // Sets downloading to false when the download finishes.
+                                    // Sets downloading to false if any errors occur.
                                     isDownloading = false
```

Comment:
Would it be better to defer `isDownload` in the `do` block instead of setting false both places?